deb-package/pkg/lib/systemd/system/unikernel.service

```diff
@@ -0,0 +1,24 @@
+[Unit]
```

Comment:
I'm thinking it's probably best if this service has a less generic name, such as `dvoting-uk.service`, to avoid conflicting with possibly other things running on our partners' VMs.